empiar/protocols/protocol_downloader.py

```diff
@@ -52,9 +52,27 @@ def _defineParams(self, form):
 
         form.addParam("gainUrl", params.StringParam, label="Gain url", help="Url in the ftp site where the gain image is")
 
+
+        ProtImportImages._defineAcquisitionParams(self, form)
+
+        line = form.addLine('Dose (e/A^2)',
+                             help="Initial accumulated dose (usually 0) and "
+                                  "dose per frame. ")
+
+        line.addParam('doseInitial', params.FloatParam, default=0,
+                      label='Initial')
+
+        line.addParam('dosePerFrame', params.FloatParam, default=None,
+                      allowsNull=True,
+                      label='Per frame')
+
         # Parallel section defining the number of threads and mpi to use
         form.addParallelSection(threads=3, mpi=1)
```

Comment:
I think this line can go away.

Reply:
Line removed :+1:

Reply:
Oh no!! I was referring to the addParallelSection